setup.py

```diff
@@ -98,15 +98,20 @@
                     os.path.relpath(root, PACKAGENAME), filename))
 package_info['package_data'][PACKAGENAME].extend(c_files)
 
+setup_requires = metadata.get('requires-dist')
+if setup_requires is None:
+    setup_requires =  ['numpy', 'cython', 'astropy']
+else:
+    # Convert the config setup_requires string into a list
+    setup_requires = setup_requires.strip().split()
 # Note that requires and provides should not be included in the call to
 # ``setup``, since these are now deprecated. See this link for more details:
 # https://groups.google.com/forum/#!topic/astropy-dev/urYO8ckB2uM
-
 setup(name=PACKAGENAME,
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['astropy'],
+      setup_requires=setup_requires,
```

Comment:
>  often numpy.get_include() is used

@cmccully - ah, but I believe that the way astropy-helpers works, if you are setting up Cython extensions you can just use ``'numpy'`` instead of ``numpy.get_include()`` and astropy-helpers does the substitution later. So I think ``setup_requires`` isn't actually needed.